### DISCUSSION TITLE
GitHub GraphQL API missing Explorer Tab

### DISCUSSION THREAD

**Discussion by VianaArthur** [community/community#24655] [Apps, API and Webhooks]
<p>Hello, I went to use the GraphQL API Explorer again and I missed the EXPLORER button, which opened a tab on the left with the references and autocomplete.<br>
Is this a bug or has it been removed?</p>
<p>Thank you for the help!</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/5/15ac40c49bec174d2ac47b116bdfb7b9b7d88254.jpeg" data-download-href="/uploads/short-url/35J4MS3ieJuxFr9VWpxcx7Y8o8A.jpeg?dl=1" title="Screen Shot 2022-01-22 at 16.38.54" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/46693700/181085867-5484c81d-2f2f-406e-aadf-544be3e94df9.jpeg" alt="Screen Shot 2022-01-22 at 16.38.54" data-base62-sha1="35J4MS3ieJuxFr9VWpxcx7Y8o8A" width="690" height="299" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/5/15ac40c49bec174d2ac47b116bdfb7b9b7d88254_2_690x299.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/1/5/15ac40c49bec174d2ac47b116bdfb7b9b7d88254_2_1035x448.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/5/15ac40c49bec174d2ac47b116bdfb7b9b7d88254_2_1380x598.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/5/15ac40c49bec174d2ac47b116bdfb7b9b7d88254_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2022-01-22 at 16.38.54</span><span class="informations">1920×832 88.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by VincentAudette** [comment]
<p>I have the same problem</p>

**Comment by Margen67** [comment]
<p>I was having this issue, but it seems to be fixed now.</p>

**Comment by VianaArthur** [comment, marked as answer]
<p>Yes, it is fixed now. Thanks!</p>

**Comment by b-mesa** [comment]
As of this writing 9/26/23 this feature has been removed once again. 
Please provide any updates if this will be restored.
Thanks

**Reply by ghostinhershell** [reply]
Hi @b-mesa, 

Thanks so much for sharing your feedback with the community. We actually made the decision to sunset the feature given it doesn't meet our accessibility standards. We opened an [issue](https://github.com/graphql/graphiql/issues/3422) with the maintainers of the repo to potentially have those accessibility concerns addressed, but in the meantime, we will not be supporting the plugin.
